core/src/Kokkos_Macros.hpp

```diff
@@ -564,6 +564,12 @@ static constexpr bool kokkos_omp_on_host() { return false; }
 
 #define KOKKOS_ATTRIBUTE_NODISCARD [[nodiscard]]
 
+#ifdef KOKKOS_ENABLE_CXX20
+#define KOKKOS_IMPL_CONDITIONAL_EXPLICIT(COND) explicit(COND)
```

Comment:
I know you like double pairs of parentheses but in case you get tired of doubling them up try
```suggestion
#define KOKKOS_IMPL_CONDITIONAL_EXPLICIT(...) explicit(__VA_ARGS__)
```

Reply:
Fixing it